internal/pkg/cli/pipeline_init.go

```diff
@@ -461,6 +319,45 @@ func (o *initPipelineOpts) selectEnvironment() error {
 	return nil
 }
 
+func (o *initPipelineOpts) askRepoDetails() error {
+	var err error
+	if o.repoURL == "" {
+		if err = o.fetchRepoURLs(); err != nil {
+			return err
+		}
+		if err = o.selectGitHubURL(); err != nil {
+			return err
+		}
+	}
+	if o.githubOwner, o.githubRepo, err = o.parseOwnerRepoName(o.repoURL); err != nil {
+		return err
+	}
+	if o.githubAccessToken == "" {
+		if err = o.getGitHubAccessToken(); err != nil {
+			return err
+		}
+	}
+	if o.gitBranch == "" {
+		o.gitBranch = defaultBranch
+	}
+	return nil
+}
+
+func (o *initPipelineOpts) fetchRepoURLs() error {
+	err := o.runner.Run("git", []string{"remote", "-v"}, command.Stdout(&o.buffer))
+	if err != nil {
```

Comment:
What happens if this is run in a folder that's not a git repository? Can we handle that error to surface a more descriptive suggestion?

Reply:
done!

Reply:
The error message is also preceded by `fatal: not a git repository (or any of the parent directories): .git`
so now it's:
```
fatal: not a git repository (or any of the parent directories): .git
✘ get remote repository info: exit status 128; make sure you have installed Git and are in a Git repository
```
which may be overkill but will leave as is for now